src/FieldExchanger.jl

```diff
@@ -28,18 +28,21 @@ have to be defined for the amtospheric component model type.
 function import_atmos_fields!(csf, model_sims, boundary_space, turbulent_fluxes)
     (; atmos_sim) = model_sims
 
-    # from atmos
-    if turbulent_fluxes == FluxCalculator.CombinedAtmosGrid()
+    # turbulent fluxes
+    if turbulent_fluxes isa FluxCalculator.CombinedAtmosGrid
```

Comment:
Do these fields still need to be remapped if the fluxes are a different type?

Reply:
Yes, though it's true that when we use regridding operationally, we should implement stubs that do nothing and avoid regridding of the same mesh.